testpilot/frontend/static-src/app/templates/experiment-page.js

```diff
@@ -59,10 +49,19 @@ export default `
                     </tr>
                   </table>
                 </section>
+                <section class="contributors-section">
+                  <h3 data-l10n-id="contributorsHeading">Brought to you by</h3>
+                  <ul class="contributors"></ul>
+                </section>
+                <section data-hook="measurements-container" class="measurements">
+                  <h3 data-l10n-id="measurements">Your privacy</h3>
+                  <div data-hook="measurements-html" class="measurement"></div>
+                  <a class="privacy-policy" data-l10n-id="experimentPrivacyNotice" data-hook="privacy-notice-url">You can learn more about the data collection for <span data-hook="title"></span> here.</a>
+                </section>
               </div>
             </div>
 
-            <div class="details-description">
+            <div class="git-description">
```

Comment:
Looks like we have [a failing test that expects `.details-description` to exist](https://github.com/mozilla/testpilot/blob/master/testpilot/frontend/static-src/test/views/experiment-page.js#L59)